o8g/Sets/Devil Reef/set.xml

```diff
@@ -438,7 +439,7 @@
         <property name="Encounter Set" value="Devil’s Reef" />
         <property name="Type" value="Location" />
         <property name="Traits" value="Cave. Y’ha-nthlei." />
-        <property name="Text" value="Forced – After this location is revealed: Place the set-aside black key on it.&#10;θ If investigators at this location control both the blue key and the yellow key and spend 2π clues, as a group: Take control of the black key and read Flashback XI in the Campaign Guide." />
```

Comment:
Here arkhamdb also has two different versions of the card. However, as they have the same card number, I am not sure if this is actually the case.

Reply:
There are 2 different versions of the same card with the same number. This happens with many of the Road locations in Horror in High Gear as well.

Reply:
That's what I thought. In that case the 2 clues per Investigator was correct

Reply:
ArkhamDB seems to be showing the same revealed image for both cards (18 & 19/38) which has 5 shroud, 1pi clues on it and has players spend 2pi clues. The thing is, card 19/38 has 3 shroud, 2pi clues, requires different keys and spends 3pi clues. Before this commit, the file already had the correct shroud, clue, and text values, except for the amount of clues to spend.
ArkhamDB seems to be showing the same image for each pair of Unfathomable Depths locations (revealed side) even though it has the actual text of each card, I think.

Reply:
At this point there are way too many versions we're referring to. To finalize this, there are already images for this pack on the module's site, please refer to that if there is any confusion.

Reply:
Yes. Just to clarify everything, all of the changes in this PR were made using the images from the module's site as reference.